web/documentserver-example/java/src/main/java/format/FormatManager.java

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
This manager will lose its meaning after the Format.type field is [FileType](https://github.com/ONLYOFFICE/document-server-integration/blob/feature/java-matrix-formats/web/documentserver-example/java/src/main/java/entities/FileType.java)